Resources/ServerInfo/_NF/Guidebook/Rules/13_Command.xml

```diff
@@ -22,4 +25,11 @@ You are the highest authority in the sector apart from Central Command. You must
     -- Granting docking extensions.
     -- Restricting vendor access (e.g. placing machines behind locked doors).
     -- Restricting access to station resources.
+    - The Sheriff or SR are under no obligation to offer to purchase contraband from other players. Seizure of illegal goods is always allowed under Space Law.
+    -- If contraband buy-backs or contraband hunting schemes are to be offered, how they are conducted is at the discretion of the SR or Sheriff.
+    -- Admins will not grant compensation to players for contraband which is seized by the authorities.
+    - The Sheriff is to ensure that any recruited cadets receive proper training and are supervised properly at all times.
+    -- Cadets should be working alongside the Sheriff or assigned to a suitably experienced officer who can oversee their training as an officer and in Space Law, SOP and the server rules relating to the NFSD.
+    -- Cadets refusing training, trying to farm playtime or insisting on working alone can face consequences IC or be ahelped for their actions.
```

Comment:
It _is_ active language. The only passive is "be ahelped for their actions", which is a perfectly correct use of the passive voice as it retains cadets as both subject and topic.

Reply:
Agreed above. This is just semantics. I'll reword some stuff with this one on recommendations because it might be easier for non-native speakers to understand.

Reply:
> It _is_ active language. The only passive is "be ahelped for their actions", which is a perfectly correct use of the passive voice as it retains cadets as both subject and topic.

"Cadets... can face consequences" - great, who's doing it?
"If cadets... reprimand them IC" - oh, it's me